Sources/MeiliSearch/Model/Task.swift

```diff
@@ -21,7 +21,20 @@ public struct Task: Codable, Equatable {
   public let status: Task.Status
 
   /// Type of the task.
-  public let type: String
+  //public let type: String
+    public enum Type:Codable {
+        case indexCreation
```

Comment:
The indentation is not following the guidelines of the project. Also, you can move this enum to the bottom of the file but still inside of the scope of the `public struct Task`.

`Type` is a protected word in Swift, so you can use `TaskType` instead.